index.js

```diff
@@ -162,10 +162,9 @@ class WebSocketJSONServer extends EventEmitter {
 
   }
 
-  // payload format: { type: 'messageType', data: $someJSON }
-  sendJson(ws,payload) {
+  send(ws,payload) {
```

Comment:
I thought I would need this function so I exposed it.
Now you can send binary, cars, whatever and not just json.